crates/interpreter/src/instructions/system.rs

```diff
@@ -91,7 +90,6 @@ pub fn calldatacopy<H: Host>(interpreter: &mut Interpreter<'_>, _host: &mut H) {
     let data_offset = as_usize_saturated!(data_offset);
     shared_memory_resize!(interpreter, memory_offset, len);
 
-    // Safety: set_data is unsafe function and memory_resize ensures us that it is safe to call it
```

Comment:
Can you revert this?

Reply:
the function is not unsafe, but I'll keep the comment as a note